google-ads-examples/src/main/java/com/google/ads/googleads/examples/extensions/AddGeoTarget.java

```diff
@@ -59,7 +59,7 @@ public static void main(String[] args) {
       params.feedItemId = Long.parseLong("INSERT_FEED_ID_HERE");
 
       // Optional: Specify a geoTargetConstantId.
-      params.geoTargetConstantId = GEO_TARGET_CONSTANT_ID;
+      params.geoTargetConstantId = null;
```

Comment:
Please add a `description` to the annotation on line 48 that explains that a default value will be used if not specified.